e2e-tests/src/test/resources/rtm-client-near-pos/RTMClientNearPOS.json

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
The entire example scenario directory copied without changes from examples library.